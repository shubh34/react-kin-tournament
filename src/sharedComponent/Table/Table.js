import './Table.scss';
import React from 'react';
import PropTypes from 'prop-types';

const Table = ({ children }) => (
	<table className="c-table">
		{children}
	</table>
);


Table.propTypes = {
	children: PropTypes.node.isRequired,
};

export default Table;
