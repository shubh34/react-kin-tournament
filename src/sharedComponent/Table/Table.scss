.c-table {
    width: 100%;
    table-layout: fixed;
    border-spacing: 0 8px;
}