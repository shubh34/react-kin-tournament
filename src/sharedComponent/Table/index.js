import Table from './Table';
import TableBody from './TableBody/TableBody';
import TableRow from './TableRow/TableRow';
import TableHeader from './TableHeader/TableHeader';
import TableColumn from './TableColumn/TableColumn';

Table.Body = TableBody;
Table.Row = TableRow;
Table.Column = TableColumn;
Table.Header = TableHeader;
export default Table;
