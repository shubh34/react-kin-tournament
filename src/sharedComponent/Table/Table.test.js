import React from 'react';
import { shallow } from 'enzyme';
import Table from './Table';

describe('Table', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<Table> Hie</Table>);
		expect(wrapper).toMatchSnapshot();
	});
});
