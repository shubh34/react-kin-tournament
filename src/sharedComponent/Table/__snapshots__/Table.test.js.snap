// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table renders without crashing 1`] = `
<table
  className="c-table"
>
   Hie
</table>
`;
